index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+    <link rel="stylesheet" href="./assets/fonts/Inter">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="toolbar">
+        <div class="toolbar-menu">
+            <nav class="nav-bar">
+                <div class="hamburger">
+                    <i class="fa fa-bars"></i>
+                </div>
+            </nav>
+
+        </div>
+    </header>
+    <section>
+        <div class="header">
+            <div class="header-content">
+                <h1 class="header-titles">Hey there. <br>I am Alex.</h1>
```

Comment:
- [ ] Hey @RileyManda you have done well so far but you still have some adjustments to make. Only your main title is allowed to be wrapped in an `<h1>` tag. The subtitle can be a `<p>` tag. For semantics, accessibility and SEO reasons, you can't have more than one `<h1>` tag on a page. make sure to wrap "I'M A SOFTWARE DEVELOPER" in a `<p>` tag